config.xml

```diff
@@ -127,6 +127,7 @@
         <registry-item name="Disable SmartScreen" path="HKLM:\SOFTWARE\Policies\Microsoft\Windows\System" value="EnableSmartScreen" type="DWord" data="0" />
         <registry-item name="Disable Microsoft Edge Phishing Filter" path="HKLM:\SOFTWARE\Policies\Microsoft\MicrosoftEdge\PhishingFilter" value="EnabledV9" type="DWord" data="0" />
         <registry-item name="Disable Windows Firewall (Standard Profile)" path="HKLM:\SOFTWARE\Policies\Microsoft\WindowsFirewall\StandardProfile" value="EnableFirewall" type="DWord" data="0" />
+        <registry-item name="Add ZoomIt to Windows Start" path="HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\run" value="ZoomIt" type="String" data="C:\Tools\sysinternals\ZoomIt64.exe" />
```

Comment:
This opens the tool Zoomit when Windows starts and keeps it open, that I think is not what we want. We want to automate clicking the option "Run ZoomIt when Windows starts" in Zoomit so that `Ctrl+1` always work by default (but without showing the ZoomIt program):
![Screenshot 2023-11-17 at 12 10 46](https://github.com/mandiant/flare-vm/assets/16052290/9023e39a-3a4a-4b71-bd7c-54b414f8f530)

I assume enabling this option modifies the registry or other configuration data.

Reply:
Fixed!